plugin/mouse/continuous_scrolling.talon

```diff
@@ -0,0 +1,3 @@
+tag: user.continuous_scrolling
+-
+<number_small>: user.mouse_scroll_set_speed(number_small)
```

Comment:
On the community backlog session we recommended that you add a graphical display of the presence of this feature.